packages/api/lib/publishSnsMessageUtils.js

```diff
@@ -33,7 +33,7 @@ const constructGranuleSnsMessage = (record, eventType) => {
     record,
     deletedAt: Date.now(),
   };
-  default: return {};
+  default: throw new Error('constructGranuleSnsMessage requires a valid eventType of \'Create\', \'Update\', or \'Delete\'');
```

Comment:
```suggestion
  default: throw new Error(`Invalid eventType: ${eventType}`);
```

I would suggest that the error specify the invalid eventType that was sent rather than enumerating the currently valid options, because those could change and the user may way to know what invalid value was received by the code